features/messages/impl/src/main/res/values/localazy.xml

```diff
@@ -4,6 +4,9 @@
     <item quantity="one">"%1$d room change"</item>
     <item quantity="other">"%1$d room changes"</item>
   </plurals>
+  <string name="screen_room_timeline_less_reactions">"Show less"</string>
+  <string name="screen_room_timeline_more_reactions">"%1$d more"</string>
```

Comment:
Not in this case

Reply:
Yes, you have to use a plural here. In English there is no diff, but in other languages it can be the case. We have the same [string on EA](https://github.com/vector-im/element-android/blob/develop/library/ui-strings/src/main/res/values/strings.xml#L3331), and a example [of translation which is different](https://github.com/vector-im/element-android/blob/develop/library/ui-strings/src/main/res/values-it/strings.xml#L2379). It's maybe because it's translated as `%d other`/ `%d others`, but this is OK because it has to stay short.

Reply:
:facepalm: good point!